### DISCUSSION TITLE
Unexpectedly short duration stats for a source

### DISCUSSION THREAD

**Discussion by cognivore** [plausible/analytics#834] [Self-Hosted Support]
## Day one source: Reddit

Looks fine:

![Day one source: reddit](https://user-images.githubusercontent.com/66186054/110960192-688db980-8346-11eb-9efb-179f7882cdb3.png)

## Day two source: All [more]

Notice how Reddit source now has 44 visitors with an average of 1 second visit time, whilst directs are in line with previous day's stats.

How would you explain it? Is it a bug? 

![Day two source: All (expanded)](https://user-images.githubusercontent.com/66186054/110960669-e782f200-8346-11eb-936b-e5d732f89e0d.png)

## Day three: All [more]

Now day three is completely confusing. Is plausible picking some bots up? Did some scrapers wake up and are creating noise in the stats?

![Day three source: All (expanded). Shows a table where almost each source has zero seconds average visit time.](https://user-images.githubusercontent.com/66186054/110961268-827bcc00-8347-11eb-8dea-021a6f8655ff.png)

**Comment by metmarkosaric** [comment]
thanks @cognivore! seems like you have a very high bounce rate on those days which would affect visit duration too. here are their definitions:

Bounce Rate

The percentage of visitors with a single page view. A visitor "bounces" away and leaves your site after only viewing a single page.

Visit Duration

How long time visitors spend on your site. It only shows people who visit more than one page. For those who visit one page only we default to 0 seconds.

https://plausible.io/docs/metrics-definitions
